pom.xml

```diff
@@ -2530,6 +2530,7 @@ under the License.
             <exclude>**/*.json</exclude>
             <exclude>**/banner.txt</exclude>
             <exclude>**/target/**</exclude>
+            <exclude>**/install-jdk.sh</exclude>
```

Comment:
I think we should avoid this.
Isn't enough to fetch the `install-jdk` script into a different location, not placed in the checkout directory, to avoid RAT failing?

Reply:
Ah, that's much better. I'll see if I can fetch it into the temp directory.

Reply:
maybe `~/bin` as it used to be?